src/app/pages/storage/volumes/volumes-list/volumes-list-table-config.ts

```diff
@@ -1105,6 +997,186 @@ export class VolumesListTableConfig implements EntityTableConfig {
     return actions as EntityTableAction[];
   }
 
+  private getStorageFlattenedAttachments(attachments: PoolAttachment[]): string {
+    const flattenAttachment = (obj: PoolAttachment): string => obj.attachments.map((a) => a.split(',').map((i) => `<br> - ${i}`).join()).join();
```

Comment:
We should break down this code into multiple properly named variable assignments on multiple lines so its more readable and understandable what's happening.